src/crimson/os/seastore/cache.cc

```diff
@@ -111,6 +111,36 @@ void Cache::replace_extent(CachedExtentRef next, CachedExtentRef prev)
   }
 }
 
+CachedExtentRef Cache::alloc_new_extent_by_type(
+  Transaction &t,       ///< [in, out] current transaction
+  extent_types_t type,  ///< [in] type tag
+  segment_off_t length  ///< [in] length
+)
+{
+  switch (type) {
+  case extent_types_t::ROOT:
+    assert(0 == "ROOT is never directly alloc'd");
+    return CachedExtentRef();
```

Comment:
nit, no need to return here.

Reply:
I really prefer that ever case have a return.